user-guide/monitoring.md

```diff
@@ -225,6 +225,19 @@ spec:
     component: core
 ```
 
+Now, create a service and `Mapping` to expose Grafana behind Ambassador:
+
+```yaml
+---
+apiVersion: getambassador.io/v1
+kind: Mapping
+metadata: 
+  name: grafana
+spec:     
+  prefix: /grafana/
+  service: grafana:{{GRAFANA_NAMESPACE}}
```

Comment:
@nbkrause shouldn't this be `grafana.{{GRAFANA_NAMESPACE}}` instead? I mean `.` instead of `:`

Reply:
Duh yes.

Reply:
Duh yes